src/App/Osu/OsuSongBrowser2.h

```diff
@@ -255,6 +255,8 @@ class OsuSongBrowser2 : public OsuScreenBackable
 	void onModeChange(UString text);
 	void onModeChange2(UString text, int id = -1);
 
+	void onPpCalcChange(UString text);
```

Comment:
"onPpCalcChange" is a bit extremely generic, make the function name more specific to your custom pp calc convars